resotoshell/resotoshell/shell.py

```diff
@@ -54,8 +54,12 @@ def post_request(
             # define content-type
             headers["Content-Type"] = content_type
             # sign request
+            headers.clear()
             if ArgumentParser.args.psk:
-                encode_jwt_to_headers(headers, {}, ArgumentParser.args.psk)
+                encode_jwt_to_headers(
+                    headers, {}, ArgumentParser.args.psk, expire_in=604800
```

Comment:
Why the long expire? A captured packet could be replayed for a week.

Reply:
Local experiment - not meant to checkin. reverted.